src/index.js

```diff
@@ -245,22 +245,28 @@ export default class EmojiReact extends Component {
 	}
 
 	render() {
-		const { reactions, onReaction, onEmojiClick } = this.props;
+		let { reactions, onReaction, onEmojiClick, sort } = this.props;
+		if (sort) {
+		reactions = reactions.slice(0);
+	    reactions.sort(function (a, b) {
+	      return b.count - a.count;
```

Comment:
<img width="389" alt="screen shot 2016-07-09 at 1 24 38 pm" src="https://cloud.githubusercontent.com/assets/8263298/16709222/82d694b6-45d8-11e6-925c-913ab5b5765f.png">

don't think it was actually fixed, looks like a mix of spaces and tabs